http/src/main/scala/org/http4s/blaze/http/http20/Http2StageTools.scala

```diff
@@ -0,0 +1,31 @@
+package org.http4s.blaze.http.http20
+
+import scala.annotation.tailrec
+
+
+object Http2StageTools {
+  // Request pseudo headers
+  val Method = ":method"
+  val Scheme = ":scheme"
+  val Path   = ":path"
+  val Authority = ":authority"
+
+  // Response pseudo header
+  val Status = ":status"
+  val Connection = "connection"
+  val TE = "te"
+  val ContentLength = "content-length"
+
+  def isLowerCase(str: String): Boolean = {
+    @tailrec
+    def go(i: Int): Boolean = {
+      if (i == str.length) true
+      else {
+        val ch = str.charAt(i)
+        if ('A' <= ch && ch <= 'Z') false
```

Comment:
to be very strict I should check to make sure its lower case ascii. This was done just so I could match against the special headers as Strings.